pkg/golinters/dupl/dupl.go

```diff
@@ -56,7 +57,17 @@ func New(settings *config.DuplSettings) *goanalysis.Linter {
 func runDupl(pass *analysis.Pass, settings *config.DuplSettings) ([]goanalysis.Issue, error) {
 	fileNames := internal.GetFileNames(pass)
 
-	issues, err := duplAPI.Run(fileNames, settings.Threshold)
+	var onlyGofiles []string
+	for _, name := range fileNames {
+		// Related to Windows
+		if !strings.HasSuffix(name, ".go") {
+			continue
+		}
+
+		onlyGofiles = append(onlyGofiles, name)
+	}
```

Comment:
Will this not have the same affect as mentioned in https://github.com/bombsimon/wsl/pull/156 making it not work with Cgo?

Reply:
No because here we filter on filenames from adjusted position.